app/models/feed.rb

```diff
@@ -143,17 +183,6 @@ def path
     "#{podcast&.path}/#{path_suffix}"
   end
 
-  def feed_episodes
-    include_in_feed = []
-    feed_max = display_episodes_count.to_i
-
-    filtered_episodes.each do |ep|
-      include_in_feed << ep if ep.include_in_feed?
-      break if (feed_max > 0) && (include_in_feed.size >= feed_max)
-    end
-    include_in_feed
-  end
-
```

Comment:
Is it possible to run both systems simultaneously? Or is this roll-forward-only?

Reply:
We'll keep the feed include/exclude tags fields for now.  So we could roll the db/code back if something goes wrong.

But functionally, I think we can only go forward.  Or else we could have contradictions.  Example: if your Default Feed excludes `bonus` episodes ... but then you navigate to the episode and click the checkbox to add it to the bonus feed anyways.